library/Haste/Generator/ModelData.php

```diff
@@ -56,7 +59,7 @@ public static function generate(\Model $objModel=null, $varCallable=null)
 
                 if ($objRelated == null) {
                     $arrData[$strField] = new Plain('', $strLabel, $arrAdditional);
-                } elseif ($objRelated instanceof \Model\Collection) {
+                } elseif ($objRelated instanceof Model\Collection) {
```

Comment:
Not sure, if you like an alias like `use Contao\Model\Collection as ContaoModelCollection` more than this how it is now?

Reply:
I like the current state of things better, never been a fan of aliasing 😉

Reply:
Fine for me. :)